development/sourcemap-validator.js

```diff
@@ -22,25 +22,38 @@ start().catch((error) => {
 
 async function start() {
   const targetFiles = [
-    `common-0.js`,
-    `background-0.js`,
-    `ui-0.js`,
-    `scripts/contentscript.js`,
-    // `scripts/inpage.js`, skipped because the validator can't sample the inlined `scripts/inpage.js` script
+    'background-0.js',
+    'common-0.js',
+    'content-script-0.js',
+    'ui-0.js',
+    'scripts/contentscript.js',
+    'scripts/disable-console.js',
+    'scripts/policy-load.js',
```

Comment:
Should we include the offscreen document file as well?

Reply:
We do, it's in the optional target files array below. It is not included in MV2 builds, so we only validate it conditionally